configure.ac

```diff
@@ -395,6 +395,16 @@ AC_ARG_ENABLE([tests],
 
 AM_CONDITIONAL(ENABLE_TESTS, test "$enable_tests" = "yes")
 
+AC_ARG_ENABLE(media-studio-va,
+    [AC_HELP_STRING([--enable-media-studio-va],
+        [enable being based on media studio libva @<:@default=no@:>@])],
+    [], [enable_media_studio_va="no"])
+
+if test "$enable_media_studio_va" = "yes"; then
+    AC_DEFINE([__ENABLE_HEVC_ENC_ON_STUDIO_VA__], [1],
```

Comment:
would better change it to be __ENABLE_STUDIO_VA__, there is an existed option of  BUILD_H265_ENCODER_. And if hevc mentioned in the file, please use a unified name  "H265", not HEVC.